imports/client/components/Routes.tsx

```diff
@@ -8,37 +8,41 @@ import EnrollForm from './EnrollForm';
 import FirstUserForm from './FirstUserForm';
 import HuntApp from './HuntApp';
 import HuntListPage from './HuntListPage';
+import Loading from './Loading';
 import LoginForm from './LoginForm';
 import PasswordResetForm from './PasswordResetForm';
 import ProfilePage from './ProfilePage';
-import RTCDebugPage from './RTCDebugPage';
 import RootRedirector from './RootRedirector';
-import SetupPage from './SetupPage';
 import UnauthenticatedRoute from './UnauthenticatedRoute';
 
+const SetupPage = React.lazy(() => import('./SetupPage'));
+const RTCDebugPage = React.lazy(() => import('./RTCDebugPage'));
+
 const Routes = React.memo(() => {
   useDocumentTitle('Jolly Roger');
 
   return (
     <BreadcrumbsProvider>
-      <Switch>
-        {/* Index redirect */}
-        <Route exact path="/" component={RootRedirector} />
+      <Suspense fallback={<Loading />}>
```

Comment:
Does putting `Suspense` here mean that if we were to e.g. make other things (like, say, Select2) dynamically load, it'd block the whole Route from rendering instead of just the thing we're trying to lazy-load?

https://reactjs.org/docs/react-api.html#reactsuspense suggests that:

> The best practice is to place `<Suspense>` where you want to see a loading indicator, but to use `lazy()` wherever you want to do code splitting.

So I'm suspicious of using Suspense way down in the root of the Routes page.  Maybe it makes more sense to inline a `<Suspense>` under the `component=` for each of the two lazy-loaded routes?

Reply:
That's where the [React docs](https://reactjs.org/docs/code-splitting.html#route-based-code-splitting) suggest putting the `Suspense` component if you're splitting by route. I think if we make more things load selectively, I'd add additional `Suspense` components further in the component tree - my assumption is that you can nest them and the innermost one gets the trigger?

Reply:
Fair enough.  I guess if that turns out not to be true, we'll find out when we get there, and in the meantime this is a win.